systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaST.java

```diff
@@ -1221,6 +1240,195 @@ void testResizeJbodVolumes() {
+     *
+     * @steps
+     *  1. - Setup environment prerequisites and configure test storage.
+     *     - Ensure the environment is in KRaft mode.
+     *  2. - Create necessary Kafka resources with additional volumes for secrets and config maps.
+     *     - Resources are correctly instantiated with specified volumes.
+     *  3. - Deploy Kafka, Kafka Connect, and Kafka Bridge with these volumes.
+     *     - Components are correctly configured with additional volumes.
+     *  4. - Verify that all pods (Kafka, Connect, and Bridge) have additional volumes mounted and accessible.
+     *     - Volumes are correctly mounted and usable within pods.
+     *
+     * @usecase
+     *  - additional-volumes
+     *  - secrets-management
+     *  - configuration-management
+     */
+    @ParallelNamespaceTest
+    @Tag(CONNECT)
+    @Tag(BRIDGE)
+    void testAdditionalVolumes() {
+        assumeTrue(Environment.isKRaftModeEnabled());
+
+        final TestStorage testStorage = new TestStorage(ResourceManager.getTestContext());
+        final int numberOfKafkaReplicas = 3;
+        final String configMapName = "example-configmap";
+        final String secretName = "example-secret";
+        final String secretKey = "apikey123456";
+        final String secretValue = "password123";
+        final String secretKeyBase64 = Base64.getEncoder().encodeToString(secretKey.getBytes());
+        final String secretValueBase64 = Base64.getEncoder().encodeToString(secretValue.getBytes());
+        final String secretMountPath = "/mnt/secret-volume";
+
+        final String configMapKey = "ENV_VAR_EXAMPLE";
+        final String configMapValue = "VALUE";
+        final String configMapMountPath = "/mnt/configmap-volume";
+
+        resourceManager.createResourceWithWait(
+                NodePoolsConverter.convertNodePoolsIfNeeded(
+                        KafkaNodePoolTemplates.brokerPoolPersistentStorage(testStorage.getNamespaceName(), testStorage.getBrokerPoolName(), testStorage.getClusterName(), numberOfKafkaReplicas).build(),
+                        KafkaNodePoolTemplates.controllerPoolPersistentStorage(testStorage.getNamespaceName(), testStorage.getControllerPoolName(), testStorage.getClusterName(), 3).build()
+                )
+        );
+
+        SecretUtils.createSecret(testStorage.getNamespaceName(), secretName, secretKeyBase64, secretValueBase64);
+        ConfigMapUtils.createConfigMap(
+                testStorage.getNamespaceName(),
+                configMapName,
+                configMapKey,
+                configMapValue);
+
+        AdditionalVolume[] additionalVolumes = new AdditionalVolume[]{
+                // Secret additional volume
+                new AdditionalVolumeBuilder()
+                        .withName(secretName)
+                        .withSecret(new SecretVolumeSourceBuilder()
+                                .withSecretName(secretName)
+                                .build())
+                        .build(),
```

Comment:
```suggestion
                        .withName(secretName)
                        .withSecret(new SecretVolumeSourceBuilder()
                              .withSecretName(secretName)
                              .build())
                        .build(),
```
applies for all other indents as well.
Check that you have something like `4,4,4` in the indent configuration of your IDE.

Reply:
Thanks...new laptop 🤷  :D